ayon_server/settings/postprocess.py

```diff
@@ -31,12 +32,24 @@ async def process_enum(
     if context is None:
         context = {}
 
+    # enum_resolver could use partial for passing arguments in
+    partial_args = []
```

Comment:
I don't think we should support positional arguments here.

Reply:
I'll check if it is still needed after change to partial function declaration.

Reply:
Not needed anymore, removed.